packages/protocol/contracts/interfaces/IProviderStore.sol

```diff
@@ -35,18 +31,13 @@ interface IProviderStore is IRequesterStore {
 
     function createProvider(
         address admin,
-        string calldata xpub
+        string calldata xpub,
+        address[] calldata authorizers
         )
         external
         payable
         returns (bytes32 providerId);
 
-    function updateProvider(
```

Comment:
Not sure I understand this. Does this mean that providers can't update their list of authorizers?

Reply:
The provider can't update the provider admin or the authorizers. Those two can only be updated by updating `config.json`.

Reply:
The idea is that `providerAdmin` is not for updating the `authorizers` of a node, but only for interacting authorizer contracts (i.e. the ChainAPI one)